air_quality_monitoring/raqmn/raqmn_data/templates/raqmn_data/plot.html

```diff
@@ -35,8 +35,8 @@
 	{% endfor %}
 	var graph = new Rickshaw.Graph({
```

Comment:
This is a beginning towards trying out other Graph APIs and raqmn_data App will serve as front end of our Project. So this is first step towards it.